stablehlo/transforms/VhloToVersion.cpp

```diff
@@ -0,0 +1,211 @@
+/* Copyright 2022 The StableHLO Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include <climits>
+#include <memory>
+#include <utility>
+
+#include "llvm/ADT/StringRef.h"
+#include "llvm/Support/Debug.h"
+#include "mlir/IR/Diagnostics.h"
+#include "mlir/IR/PatternMatch.h"
+#include "mlir/Support/LogicalResult.h"
+#include "mlir/Transforms/DialectConversion.h"
+#include "stablehlo/dialect/Version.h"
+#include "stablehlo/dialect/VhloOps.h"
+#include "stablehlo/transforms/Passes.h"
+#include "stablehlo/transforms/TypeConversion.h"
+
+#define DEBUG_TYPE "compat-passes"
+
+namespace mlir {
+namespace stablehlo {
+#define GEN_PASS_DEF_VHLOTOVERSIONPASS
+#include "stablehlo/transforms/Passes.h.inc"
+}  // namespace stablehlo
+
+///////////////////////
+/// VHLO To Version ///
+///////////////////////
+namespace vhlo {
+namespace {
+using namespace vhlo;  // For VHLO Ops, Version, VhloDialect.
```

Comment:
We are already in `namespace vhlo`, right?

Reply:
Whoops. I moved this all to `namespace vhlo` to explicitly get rid of the `using namespace` declaration. Had to shuffle things around to move passes to `namespace stablehlo`.